### PR TITLE
Modifying image asset documentation

### PR DESCRIPTION
**Pull request by eliserichards**
Fixes N/A

Adding some notes to our image asset and architecture documentation.

## To Do

- [ ] add “WIP” to the PR title if pushing up but not complete nor ready for review
- [ ] double check the original issue to confirm it is fully satisfied

### CODE REVIEW

File: docs/add_image_assets.md [hunk 1]
```diff
@@ -5,9 +5,15 @@ Lockbox app uses 2 types of assets for icons/images that are added in the projec
...
-- To open Vector Asset Studio tool, go to app->res->drawable folder in Android View, right click on drawable folder, and select New -> Vector Asset
-- Vector Asset Studio opens. Select Local file, specify the .svg image file path, override the Size and optionally change icon name. Then select Next and Finish
-- More information related to Vector Drawables and Vector Asset Studio can be found [here](https://developer.android.com/studio/write/vector-asset-studio)
-- If the image is more complex, has a larger size, or the vector drawable contains attributes that are not supported starting from the minSdk level set in the project (currently minSdk api level is 23), like for example fillType or gradient attributes, we should consider adding flat images for different resolutions. The .png files are also available in Zeplin and they have to be added in res->drawable specific folders: drawable-mdpi, drawable-hdpi, drawable-xhdpi, drawable-xxhdpi.
+  - Vector Assets are more appropriate for simple icons, that are not very detailed and do not have a large size (documentation recommends a maximum of 200 x 200 dp).
```
Comment (jhugman) [inline docs/add_image_assets.md]: The comma confused me, perhaps:

> Simple icons with few details which are intended for small display size (max 200x200dp [link to documentation]) should use vector assets.

File: docs/add_image_assets.md [hunk 2]
```diff
@@ -5,9 +5,15 @@ Lockbox app uses 2 types of assets for icons/images that are added in the projec
...
+  - Vector Assets should be added to one the following folders in the application:
+    - pp->res->drawable->drawable: for simple icons from Android libraries
+    - pp->res->drawable->drawable-v24: for more complex images (e.g. the Lockbox logo)
+  - To open Vector Asset Studio tool, go to the appropriate folder in Android View, right click on drawable folder, and select New -> Vector Asset
+  - Vector Asset Studio opens. Select Local file, specify the .svg image file path, override the Size and optionally change icon name. Then select Next and Finish
+  - More information related to Vector Drawables and Vector Asset Studio can be found [here](https://developer.android.com/studio/write/vector-asset-studio)
+- If the image is more complex, has a larger size, or the vector drawable contains attributes that are not supported starting from the minSdk level set in the project (currently minSdk api level is 23), like for example fillType or gradient attributes, we should consider adding flat images for different resolutions.
```
Comment (jhugman) [inline docs/add_image_assets.md]: Use back ticks around `fillType` and `gradients`.

File: docs/add_image_assets.md [hunk 3]
```diff
@@ -5,9 +5,15 @@ Lockbox app uses 2 types of assets for icons/images that are added in the projec
...
+  - The .png files are also available in Zeplin and they have to be added in res->drawable specific folders: drawable-mdpi, drawable-hdpi, drawable-xhdpi, drawable-xxhdpi.
+  - Download the .png files and add each of the images to their corresponding res->drawable folder in Android View
+  - If you wish to add both a Vector Asset and .png assets to your project, ensure that all versions of the same image have the same file name (e.g. "ic_default_avatar").
```
Comment (jhugman) [inline docs/add_image_assets.md]: Use backticks around `ic_default_avatar` instead of double quotes.

File: docs/add_image_assets.md [hunk 4]
```diff
@@ -5,9 +5,15 @@ Lockbox app uses 2 types of assets for icons/images that are added in the projec
...
+    - pp->res->drawable->drawable: for simple icons from Android libraries
+    - pp->res->drawable->drawable-v24: for more complex images (e.g. the Lockbox logo)
+  - To open Vector Asset Studio tool, go to the appropriate folder in Android View, right click on drawable folder, and select New -> Vector Asset
+  - Vector Asset Studio opens. Select Local file, specify the .svg image file path, override the Size and optionally change icon name. Then select Next and Finish
+  - More information related to Vector Drawables and Vector Asset Studio can be found [here](https://developer.android.com/studio/write/vector-asset-studio)
+- If the image is more complex, has a larger size, or the vector drawable contains attributes that are not supported starting from the minSdk level set in the project (currently minSdk api level is 23), like for example fillType or gradient attributes, we should consider adding flat images for different resolutions.
+  - The .png files are also available in Zeplin and they have to be added in res->drawable specific folders: drawable-mdpi, drawable-hdpi, drawable-xhdpi, drawable-xxhdpi.
```
Comment (jhugman) [inline docs/add_image_assets.md]: `Backticks` around `code` like `things` :)

**Comment by devinreams** [comment]
Thanks for helping keep our docs awesome, @eliserichards!

(and reviewers 😄)

**Pull request merged**